nextcord/message.py

```diff
@@ -361,6 +363,14 @@ def to_dict(self) -> AttachmentPayload:
             result["description"] = self.description
         return result
 
+    @property
```

Comment:
https://github.com/nextcord/nextcord/pull/1083#discussion_r1258078471

Might also be good for some helper functions/properties instead, like `Role.is_bot_managed`, since `message.flags.is_remix` is not really needed.